crates/sui-rpc-api/proto/sui.node.v2.proto

```diff
@@ -0,0 +1,363 @@
+syntax = "proto3";
+
+package sui.node.v2;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+import "sui.types.proto";
+
+//
+// Rpc Node interface
+//
+
+service Node {
+  rpc HealthCheck(HealthCheckRequest) returns (google.protobuf.Empty);
```

Comment:
Yep and i think we should also support that. But unfortunately that health checking service doesn't really expose the ability to have the type of "health check" that we want here. This one here allows you to ask how far behind in syncing the node is vs the gRPC service which is really just a pulse check. Maybe this needs to be better named?

Reply:
I see, makes sense. In that case +1 for renaming this to prevent others from being similarly confused about whether this is trying to replicate the standard health checking mechanism

Reply:
For now i just removed it.